### ISSUE TITLE
`for await` is not supported to be transformed

### ISSUE THREAD

**Issue by JounQin** [MatAtBread/fast-async#70]
As title.

```ts
// source code
const main = async () => {
  for await (const i of [1, 2, 3]) {
    console.log(i);
  }
};

// compiled code
const main = () => new Promise(function ($return, $error) {
  for await (const i of [1, 2, 3]) {
    console.log(i);
  }
};
```

---

Is this project dead?

**Comment by Rush** [comment]
I think this project lost relevance due to better native implementations of async/await.

**Comment by MatAtBread** [comment]
Yes, it's pretty much dead. Most runtimes support async/await natively now, and I have no requirement to support legacy ones myself. If you want to fork & fix, go ahead

**Comment by JounQin** [comment]
> Yes, it's pretty much dead. Most runtimes support async/await natively now, and I have no requirement to support legacy ones myself. If you want to fork & fix, go ahead

Thank you, that's fine. Can you document it or just achieve this repository?
